content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md

```diff
@@ -1177,6 +1183,33 @@ The `fieldPath` field does not support indexing arrays numerically.
 
 Setting `fieldPath` is optional.
 
+#### The `optionalOldSelf` field {#field-optional-oldself}
+
+The `optionalOldSelf` field is a boolean field added in Kubernetes 1.29. The feature
+[CRDValidationRatcheting](#validation-ratcheting) must be enabled in order to 
+make use of this field.
+
+This field alters the behavior of [Transition Rules](#transition-rules) described
+below. Normally, a transition rule will not evaluate if `oldSelf` cannot be determined:
+during object creation or when a new value is introduced in an update.
+
+If `optionalOldSelf` is set to true, then transition rules will always be 
+evaluated and the type of `oldSelf` be changed to a CEL [`Optional`](https://pkg.go.dev/github.com/google/cel-go/cel#OptionalTypes) type. 
+
+`optionalOldSelf` is useful in cases where schema authors would like a more
+powerful tool than [implicit deepequal validation ratcheting][#validation-ratcheting] 
```

Comment:
```suggestion
control than provided by the default equality based behavior of [validation ratcheting][#validation-ratcheting] 
```

?

(Users might not know what we mean by "deepequal")

Reply:
added